src/__tests__/auth_Test/resetActions.test.js

```diff
@@ -0,0 +1,80 @@
+import MockAdapter from "axios-mock-adapter";
+import configureMockStore from "redux-mock-store";
+import InvokeReset, {
+  newPasswordRequest
+} from "../../actions/authActions.js/passswordResetAction";
+import { simpleAction } from "../../actions/simpleAction";
+import { SIMPLE_ACTION } from "../../actions/actionTypes";
+
+import axios from "axios";
+
+const data = {
+  user: {
+    email: "admin@admin.com",
+    url: "https://ah-backend-lannister-staging.herokuapp.com/reset_confirm/"
+  }
+};
+const responseData = {
+  user: {
+    message: "a reset link has been sent to your email admin@admin.com"
+  }
+};
+const REQUEST_RESET_URL =
+  "https://ah-backend-lannister-staging.herokuapp.com/api/users/password_reset";
```

Comment:
@kimbugp kindly store the base URL "https://ah-backend-lannister-staging.herokuapp.com" in a variable BASE_URL to avoid rewriting it every time you are making a request.

Reply:
Thanks Miriam,I have set it up